awx/ui_next/src/screens/User/shared/UserForm.jsx

```diff
@@ -145,8 +145,8 @@ function UserForm({ user, handleCancel, handleSubmit, submitError, i18n }) {
       if (!values.password || values.password === '') {
         delete values.password;
       }
-      delete values.confirm_password;
```

Comment:
If the save fails, the form will attempt to reload the deleted value. Since it was deleted, it'll be undefined and trigger a validation error for being missing.